api/src/controllers/website.js

```diff
@@ -99,9 +99,35 @@ async function updatePublicationOptions(req, res, next) { // eslint-disable-line
   }
 }
 
+/**
+ * Update website title on the deployed website
```

Comment:
Could you elaborate on what title we're deploying here, it's not very clear from just reading the docs.

Maybe include a line saying `The title is the HTML <title> tag used for client websites. Currently, we are using the team's name for the title, but this route can be used to update title if need be`/similar.